AWS-CREDENTIALS.md

```diff
@@ -77,19 +79,19 @@ let auto_refreshing_provider = rusoto_credential::AutoRefreshingProvider::new(pr
 
 #### Credential refreshing
 
-Credentials obtained from environment variables and credential files expire ten minutes after being acquired and are refreshed on subsequent calls to `credentials()` (a method from the `ProvideAwsCredentials` trait).
+Credentials obtained from environment variables and credential files expire **ten minutes** after being acquired and are refreshed on subsequent calls to `credentials()`, a method from the `ProvideAwsCredentials` trait.
```

Comment:
We should update this old info: [credential files don't reload](https://github.com/rusoto/rusoto/issues/1303).

Reply:
we should!

Reply:
Are you able to do this in this PR or should we make another?

Reply:
I'm just going to remove this for now. Incorrect docs are worse that incomplete docs